test/api_tests.jl

```diff
@@ -20,3 +20,55 @@ pos1 = model1.space.agent_positions[coord2vertex((1,1), model1)]
 @test length(pos1) == 0
 pos2 = model1.space.agent_positions[coord2vertex((2,2), model1)]
 @test pos2[1] == 1
+
```

Comment:
It is worth putting tests inside a `@testset` block.

Reply:
I am on it!